### ISSUE TITLE
Error in online documentation

### ISSUE THREAD

**Issue by jorgensd** [FEniCS/ufl#106]
The docstrings of [ufl.operators.dot](https://fenics.readthedocs.io/projects/ufl/en/latest/api-doc/ufl.html#ufl.operators.dot) does not match the actual [documentation](https://github.com/FEniCS/ufl/blob/main/ufl/operators.py#L171) of the function,
i.e.

> UFL operator: Take the dot product of a and b. The complex conjugate of the second argument is taken.
and 

> "UFL operator: Take the dot product of *a* and *b*. This won't take the complex conjugate of the second argument."

**Comment by mscroggs** [comment]
This is correct the latest version of the docs at https://docs.fenicsproject.org/ufl/2022.2.0/api-doc/ufl.html#ufl.operators.dot.

Looks like the readthedocs link is old? We should either delete that or make it build again
